mentors/ruby/t.json

```diff
@@ -23,4 +23,12 @@
     "avatar_url": null,
     "bio": "Hi, there! I've been working with Ruby since 2015 and am a big fan of the community and mentorship. My previous career was in Music/Education and I find those skills have helped me be a better teammate and teacher. I look forward to working with you!"
   }
+  {
+  "github_username": "thipbpoullain",
+  "name": "Thibaut Poullain",
+  "link_text": null,
+  "link_url": null,
+  "avatar_url": null,
+  "bio": "I fell in love with Ruby and her community 5 years ago. Today I use Ruby every day as a freelancer. I now have the opportunity to give everything the community gave me!"
+}
```

Comment:
```suggestion
    "github_username": "thipbpoullain",
    "name": "Thibaut Poullain",
    "link_text": null,
    "link_url": null,
    "avatar_url": null,
    "bio": "I fell in love with Ruby and her community 5 years ago. Today I use Ruby every day as a freelancer. I now have the opportunity to give everything the community gave me!"
  }
```
These lines should be indented in the same fashion as the entries before this one.